src/services/WebWorker.js

```diff
@@ -43,6 +43,7 @@ export default class WebWorker {
             resolve(event.data.result)
           } else {
           }
```

Comment:
can we just remove? i think equally as confusing now that there's a todo. it seems like it does nothing?

Reply:
yeah added todo to get a comment from you :). will remove